### ISSUE TITLE
Migrate backend to Firebase

### ISSUE THREAD

**Issue by hanmajid** [hanmajid/flutter_whatsapp#4]
### Current behavior

Backend is personal dummy server.

### Expected behavior

Migrate backend to Firebase (Firestore).

**Comment by Mastersam07** [comment]
Is this currently worked upon?

**Comment by hanmajid** [comment]
Yup. Although I'm prioritizing migrating to AndroidX first (https://github.com/hanmajid/flutter_whatsapp/issues/2). 🙂

**Comment by Mastersam07** [comment]
Saw that issue. Trying to replicate here.

**Comment by hanmajid** [comment]
Yeah, this repo's code is pretty old. So there's a lot of things that need fixing. Feel free to report issues or create pull requests if you want. Anything is very much appreciated. 🙂
